### DISCUSSION TITLE
Configuration course not show options buttons

### DISCUSSION THREAD

**Discussion by eanunezt** [chamilo/chamilo-lms#5248] [General]
When creating a course and then going to the course configuration option, the options to select yes or no or to upload files do not work, you only see an x ​​in parentheses. Do I need something to do? the version is: 1.11.26
![image](https://github.com/chamilo/chamilo-lms/assets/31119578/60f7b255-b2ac-4bb4-b10a-fa4275388196)

**Comment by contidos** [comment]
Is it a campus with multiurl configuration?
